### PR TITLE
[17.0][OU-ADD] iap: Nothing to do

### PR DESCRIPTION
**Pull request by acpMicrocom**

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]
OK for me. I would say, please just squash the 3 commits in 1.

**Review by xtanuiha — APPROVED** [review]

**Comment by legalsylvain** [comment]
/ocabot migration iap

<dependency></dependency>

File: docsource/modules160-170.rst [hunk 1]
```diff
@@ -204,7 +204,7 @@ Module coverage 16.0 -> 17.0
 +---------------------------------------------------+----------------------+-------------------------------------------------+
 | http_routing                                      |                      |                                                 |
 +---------------------------------------------------+----------------------+-------------------------------------------------+
-| iap                                               |                      |                                                 |
+| iap                                               | Done                 |                                                 |
```
Comment (legalsylvain) [inline docsource/modules160-170.rst:207]:
```suggestion
| iap                                               | Nothing to do        |                                                 |
```

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Lgtm. Except state of module

**Comment by remi-filament** [comment]
Hi @legalsylvain thanks for taking over this PR can you please close this PR and update migration issue with #4689 merged ?

**Pull request closed without merging**